src/azure-cli-core/azure/cli/core/util.py

```diff
@@ -66,88 +66,110 @@ def handle_exception(ex):  # pylint: disable=too-many-locals, too-many-statement
 
     error_msg = getattr(ex, 'message', str(ex))
     exit_code = 1
+    az_error = None
+
+    command_string = None
+
+    if command:
+        command_string = " ".join(command)
 
     if isinstance(ex, azclierror.AzCLIError):
         az_error = ex
+        az_error.request_error_assistance(cli_ctx)
 
     elif isinstance(ex, JMESPathError):
         error_msg = "Invalid jmespath query supplied for `--query`: {}".format(error_msg)
-        az_error = azclierror.InvalidArgumentValueError(error_msg)
+        az_error = azclierror.InvalidArgumentValueError(error_msg, command=command_string)
+        az_error.request_error_assistance(cli_ctx)
```

Comment:
Could we add `az_error.request_error_assistance(cli_ctx)` only once for all cases at the end of the entire method?